frontend/src/lib/constants.tsx

```diff
@@ -68,13 +68,13 @@ export const PROPERTY_MATH_TYPE = 'property'
 export const EVENT_MATH_TYPE = 'event'
 export const MATHS: Record<string, any> = {
     total: {
-        name: 'Total volume',
+        name: 'Total count',
         description: (
             <>
-                Total event volume.
+                Total event volume count. Number of times the user performed the event.
```

Comment:
nits, 2c on the copy: 
- I suggest we say either Total event volume or Total event count but not event volume count.

Maybe in the first sentence of the description here we shouldn't mention users since they aren't really tied to volume? So just: 

'Total event count. Number of times the event was performed in a given period.'

and then retain the example you have using the User that performed the event 3 times.